storm-core/src/jvm/org/apache/storm/cluster/Cluster.java

```diff
@@ -0,0 +1,236 @@
+import org.apache.storm.generated.ExecutorInfo;
+import org.apache.storm.generated.ExecutorStats;
+import org.apache.storm.generated.ProfileAction;
+import org.apache.storm.utils.Utils;
+import org.apache.zookeeper.ZooDefs;
+import org.apache.zookeeper.data.ACL;
+import org.apache.zookeeper.data.Id;
+import org.apache.zookeeper.server.auth.DigestAuthenticationProvider;
+
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Cluster {
+
+    public static final String ZK_SEPERATOR = "/";
+
+    public static final String ASSIGNMENTS_ROOT = "assignments";
+    public static final String CODE_ROOT = "code";
+    public static final String STORMS_ROOT = "storms";
+    public static final String SUPERVISORS_ROOT = "supervisors";
+    public static final String WORKERBEATS_ROOT = "workerbeats";
+    public static final String BACKPRESSURE_ROOT = "backpressure";
+    public static final String ERRORS_ROOT = "errors";
+    public static final String BLOBSTORE_ROOT = "blobstore";
+    public static final String BLOBSTORE_MAX_KEY_SEQUENCE_NUMBER_ROOT = "blobstoremaxkeysequencenumber";
+    public static final String NIMBUSES_ROOT = "nimbuses";
+    public static final String CREDENTIALS_ROOT = "credentials";
+    public static final String LOGCONFIG_ROOT = "logconfigs";
+    public static final String PROFILERCONFIG_ROOT = "profilerconfigs";
+
+    public static final String ASSIGNMENTS_SUBTREE;
+    public static final String STORMS_SUBTREE;
+    public static final String SUPERVISORS_SUBTREE;
+    public static final String WORKERBEATS_SUBTREE;
+    public static final String BACKPRESSURE_SUBTREE;
+    public static final String ERRORS_SUBTREE;
+    public static final String BLOBSTORE_SUBTREE;
+    public static final String BLOBSTORE_MAX_KEY_SEQUENCE_NUMBER_SUBTREE;
+    public static final String NIMBUSES_SUBTREE;
+    public static final String CREDENTIALS_SUBTREE;
+    public static final String LOGCONFIG_SUBTREE;
+    public static final String PROFILERCONFIG_SUBTREE;
+
+    static {
+        ASSIGNMENTS_SUBTREE = ZK_SEPERATOR + ASSIGNMENTS_ROOT;
+        STORMS_SUBTREE = ZK_SEPERATOR + STORMS_ROOT;
+        SUPERVISORS_SUBTREE = ZK_SEPERATOR + SUPERVISORS_ROOT;
+        WORKERBEATS_SUBTREE = ZK_SEPERATOR + WORKERBEATS_ROOT;
+        BACKPRESSURE_SUBTREE = ZK_SEPERATOR + BACKPRESSURE_ROOT;
+        ERRORS_SUBTREE = ZK_SEPERATOR + ERRORS_ROOT;
+        BLOBSTORE_SUBTREE = ZK_SEPERATOR + BLOBSTORE_ROOT;
+        BLOBSTORE_MAX_KEY_SEQUENCE_NUMBER_SUBTREE = ZK_SEPERATOR + BLOBSTORE_MAX_KEY_SEQUENCE_NUMBER_ROOT;
+        NIMBUSES_SUBTREE = ZK_SEPERATOR + NIMBUSES_ROOT;
+        CREDENTIALS_SUBTREE = ZK_SEPERATOR + CREDENTIALS_ROOT;
+        LOGCONFIG_SUBTREE = ZK_SEPERATOR + LOGCONFIG_ROOT;
+        PROFILERCONFIG_SUBTREE = ZK_SEPERATOR + PROFILERCONFIG_ROOT;
+    }
```

Comment:
I would prefer to see these inlined above, instead of in a static block.

for example

```
public static final String ASSIGNMENTS_SUBTREE = ZK_SEPERATOR + ASSIGNMENTS_ROOT;
```